src/main/java/com/datalogics/pdf/samples/forms/ExportFormDataToCsv.java

```diff
@@ -155,26 +156,48 @@ private static void exportFieldValues(final PDFInteractiveForm form, final CSVPr
         while (fieldIterator.hasNext()) {
             final PDFField field = fieldIterator.next();
 
-            final String formattedValue = field.getFormattedValue();
-            if (formattedValue == null) {
-                final List value = field.getValueList();
-                if (value == null) {
-                    printer.print("");
-                    if (LOGGER.isLoggable(Level.WARNING)) {
-                        LOGGER.warning(field.getQualifiedName() + " has no value!");
-                    }
+            List value;
+            if (VersionUtils.pdfjtIsBeforeVersion4()) {
+                value = field.getValueList();
+                validateField(printer, field, value);
+            } else {
+                final String formattedValue = field.getFormattedValue();
```

Comment:
@datalogics-coreyy, I didn't have anything specific in mind. Thinking about it a little more though and having a method for each version could end up with a large number of methods. Maybe having a method where the sample can pass a version string to test would be a different solution to consider? That way there is one method additional method and samples could modify their behavior granularly.